### PR TITLE
Update to add IISAdministration powershell gallery information

### PR DESCRIPTION
**Pull request by jhkimnew**
@bariscaglar Would you review this doc change? 

This is for #105.

### CODE REVIEW

File: iis/get-started/whats-new-in-iis-10/iisadministration-powershell-cmdlets.md [hunk 1]
```diff
@@ -21,9 +21,10 @@ by Baris Caglar
 
 | Version | Notes |
 | --- | --- |
-| IIS 10.0 | IISAdministration PowerShell Cmdlets were introduced in IIS 10.0 |
-| IIS 8.5 and earlier | IISAdministration PowerShell Cmdlets were not supported prior to IIS 10.0 |
+| IIS 10.0 | IISAdministration PowerShell Cmdlets were introduced in IIS 10.0. The latest version of IISAdministration is supported via Powershell gallery. |
+| IIS 8.5 and earlier | IISAdministration PowerShell Cmdlets were not supported prior to IIS 10.0. However users can install it via Powershell gallery. |
 
+Visit [iis.net](https://blogs.iis.net/iisteam/introducing-iisadministration-in-the-powershell-gallery "Introducing IISAdministration in the Powershell gallery") to know about how to get the latest version of IISAdministration.
```
Comment (bariscaglar) [inline iis/get-started/whats-new-in-iis-10/iisadministration-powershell-cmdlets.md]: "to learn about" would be better here.

Comment (jhkimnew) [reply iis/get-started/whats-new-in-iis-10/iisadministration-powershell-cmdlets.md]: Okay, I have update it.

**Comment by bariscaglar** [comment]
You can go ahead and close the issue.

**Comment by jhkimnew** [comment]
Done.

**Pull request merged**